### ISSUE TITLE
mat-select doesn't change appearance when focused with keyboard

### ISSUE THREAD

**Issue by mertdeg2** [angular/material2#3748]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
mat-select is highlighted when user tabs into the component

#### What is the current behavior?
mat-select doesn't change its appearance when focused with keyboard

#### What are the steps to reproduce?
Press "Tab" key until the keyboard focus is on mat-slide-toggle

Btw. default plunkr template for creating bugs seem to be not working: https://goo.gl/DlHd6U

**Comment by kara** [comment]
@mertdeg2 I'm seeing the correct focus styles when TABBing on the docs site:

https://material.angular.io/components/component/select

To clarify, what do you expect to see?

**Comment by kara** [comment]
Or are you talking about #3743 ?

**Comment by jelbourn** [comment]
Closing this since it's marked as "Cannot reproduce" and possibly was fixed as part of #3743

**Comment by thornblom** [comment]
@kara I have the same problmen, and not seeing the correct focus styles on docs site. I can navigare and select correct mat-option with UP/DOWN key, but I can't see what is focus due to no appearance change.

**Comment by pbiradar0** [comment]
hello, 

I'm also facing the same issue, here is the scenario,

We want to focus select element using the tab option, I tried below things but didn't work,

HTML:

<div>
				<label>
					{{ 'UserSettings_Lable_Selectlanguage' | optTranslate }}
				</label>
				<opt-select
					#ddlLanguage
					elementId="ddlLanguage"
					[options]="lanList"
					text="DisplayName"
					controlId="language"
					value="Id"
					tooltip="DisplayName"
					[disableOption]="false"
					[selectForm]="userDetailForm"
				>
				</opt-select>
			</div>


TS:

@ViewChild('ddlLanguage') public ddlLanguage: MatSelect;
onInit(){

this.ddlLanguage.focus(); //Also tried with https://stackoverflow.com/questions/54593925/how-to-set-auto-focus-in-mat-select

}

Thanks in advanced.
